examples/default.yaml

```diff
@@ -61,11 +61,28 @@ mounts:
     # system will look and feel like regular files directories in the Guest OS.
     # 🟢 Builtin default: false
     followSymlinks: null
+  9p:
+    # Supported security models are "passthrough", "mapped-xattr", "mapped-file" and "none".
+    # 🟢 Builtin default: "mapped-xattr"
+    securityModel: null
+    # Select 9P protocol version. Valid options are: "9p2000" (legacy), "9p2000.u", "9p2000.L".
+    # 🟢 Builtin default: "9p2000.L"
+    protocolVersion: null
+    # The number of bytes to use for 9p packet payload, where 4KiB is the absolute minimum.
+    # 🟢 Builtin default: "128KiB"
+    msize: null
```

Comment:
This had to be "mSize" rather than "msize"? idk.

Reply:
We discussed this at https://github.com/lima-vm/lima/pull/726#discussion_r840968697, and @afbjorklund decided to use it as an "opaque name", so would be `Msize` and `msize` respectively.

Reply:
I didn't like mSize, even if "Msize" isn't ideal (in Go)